packages/dashboard/src/project/hook/hookEdit.tsx

```diff
@@ -64,53 +79,125 @@ export const HookEdit = ({
           },
         },
       });
-
+      setDeleting(false);
       dispatch({
         type: 'REMOVE_HOOK',
         payload: {
           hookId,
         },
       });
     } catch (error) {
+      setDeleting(false);
+      let errorMessage = 'Failed to delete hook';
+      if (error instanceof Error) {
+        errorMessage = error.message;
+      }
+      setDeleteError(errorMessage);
       console.error(error);
     }
   }
   return (
     <>
-      <HFlow justifyContent="space-between" alignItems="center">
+      <Box
+        style={{ display: 'flex' }}
+        justifyContent="space-between"
+        alignItems="center"
+      >
         <Toggler
           toggleExpanded={toggleExpanded}
           isExpanded={isExpanded}
-          title={`${enumToString(hook.hookType)}: ${hook.url || 'New Hook'}`}
+          title={hookTitle}
         />
         <Button
-          kind="danger"
-          skin="ghost"
+          variant="contained"
+          color="error"
           size="small"
           disabled={disabled}
-          onClick={() => deleteHook(hook.hookId)}
+          onClick={() => setModal(true)}
         >
-          Remove
+          <DeleteIcon fontSize="small" />
         </Button>
-      </HFlow>
+      </Box>
       {isExpanded && <HookDetails hook={hook} />}
+      <DeleteModal
+        isModalActive={isModalActive}
+        handleCloseModal={() => setModal(false)}
+        handleDelete={() => deleteHook(hook.hookId)}
+        deleting={deleting}
+        hookName={hookTitle}
+        deleteError={deleteError}
+      />
     </>
   );
 };
 
 const HookDetails = ({ hook }: { hook: Hook }) => {
   return (
-    <Grid style={{ padding: '1rem' }}>
-      <Cell xs={12}>
+    <Grid sx={{ marginTop: '1rem' }}>
```

Comment:
```suggestion
    <Grid container mt={2}>
```
You don't need sx for styling components (Grid, Box, ... ).
use spacing instead of `rem`. (2 is 16px set what you need here)

Reply:
done